### ISSUE TITLE
Allow more flexibility in correcting names

### ISSUE THREAD

**Issue by tmtmtmtm** [mysociety/verification-pages#296]
On [SG13](https://www.wikidata.org/wiki/User:Verification_pages_bot/verification/sg/Member_of_the_13th_Parliament), the page won't allow me to correct "Lim, W K", to "Lim Wee Kiak", "Chong, K H" to "Chong Kee Hiong", "Ong, T K" to "Ong Teng Koon", "Seah, K P" to "Seah Kian Peng", or "Yee, C H" to "Yee Chia Hsing":

![screen shot 2018-07-18 at 10 08 32](https://user-images.githubusercontent.com/57483/42871966-8545b18c-8a73-11e8-8853-d358caa66dd2.png)

If the reason for doing this (which isn't particularly clear on the page) is that this is the label that will be set on the freshly created Wikidata Item for each person, then it seems quite important that the user be able to make corrections like these.

**Comment by tmtmtmtm** [comment]
This is affecting:
* https://www.wikidata.org/wiki/User:Verification_pages_bot/verification/sg/Member_of_the_13th_Parliament

**Comment by tmtmtmtm** [comment]
I think this is no longer relevant after #355
